README.md

```diff
@@ -12,206 +12,614 @@ A command-line tool for managing TEE deployments on Phala Network, from local de
 
+**Options:**
+- `-k, --key <key>`: API key to set (if not provided, you will be prompted)
+
+**Example:**
 ```bash
-Deploying CVM ...
-Deployment successful
-App Id: cc3ee84d7e708aed326d5df6d22296f65b4fd99e
-App URL: https://cloud.phala.network/dashboard/cvms/app_cc3ee84d7e708aed326d5df6d22296f65b4fd99e
+phala auth login --key your-phala-cloud-api-key
 ```
 
-### lightclient example
+#### Logout
 
 ```bash
-# Deploy the lightclient example
-teecloud deploy -c ./examples/lightclient/docker-compose.yml -n lightclient
+phala auth logout
 ```
 
-Example Output:
+Remove the stored API key.
 
+**Example:**
 ```bash
-Deploying CVM ...
-Deployment successful
-App Id: c4b66ce50498b3da25555030f714e6ece55d8a35
-App URL: https://cloud.phala.network/dashboard/cvms/app_c4b66ce50498b3da25555030f714e6ece55d8a35
+phala auth logout
 ```
 
-## Local Development Workflow
+#### Status
 
-### 1. Start the TEE Simulator
+```bash
+phala auth status [options]
+```
 
-Test your application in a local TEE environment:
+Check your authentication status with Phala Cloud. Displays user information in a table format.
 
+**Options:**
+- `-j, --json`: Output in JSON format
+
+**Example:**
 ```bash
-teecloud simulator
+phala auth status
+phala auth status --json
 ```
 
-The simulator will be available at http://localhost:8090.
+### TEEPod Management Commands
 
-### 2. Build Your Docker Image
+Commands for managing TEEPods on Phala Cloud.
+
+#### List TEEPods
 
 ```bash
-teecloud build \
-  -i, --image <name> \         # Required: Docker image name
-  -u, --username <name> \      # Required: Docker Hub username
-  -f, --dockerfile <path> \    # Required: Path to Dockerfile
-  -t, --tag <tag>             # Required: Tag for the Docker image
+phala teepods list
 ```
 
-### 3. Test Locally
+List all available TEEPods on Phala Cloud.
 
-Generate a compose file:
+**Example:**
 ```bash
-teecloud build-compose \
-  -i, --image <name> \         # Required: Docker image name
-  -u, --username <name> \      # Required: Docker Hub username
-  -t, --tag <tag> \           # Required: Tag for the Docker image
-  -e, --env-file <path> \      # Required: Path to environment file
-  -v, --version <version>      # Optional: Version of compose template (default: 'basic')
+phala teepods list
 ```
 
-Run locally:
+#### List TEEPod Images
+
 ```bash
-teecloud run-local \
-  -c, --compose <path> \       # Required: Path to docker-compose file
-  -e, --env-file <path>        # Required: Path to environment file
+phala teepods images [options]
 ```
 
-The CLI stores generated compose files in:
+List available images for a specific TEEPod.
+
+**Options:**
+- `-t, --teepod-id <teepodId>`: TEEPod ID (required)
+
+**Example:**
+```bash
+phala teepods images --teepod-id 2
 ```
-.tee-cloud/
-  └── compose-files/     # Generated docker-compose files
-      └── tee-compose.yaml
+
+### Docker Management Commands
```

Comment:
I recommend moving related stuff to another doc file to make README clean

Reply:
Updated README.md. Let me know how this looks